com.github.TuxPusher.yaml

```diff
@@ -0,0 +1,25 @@
+app-id: com.github.TuxPusher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: tuxpusher
+rename-icon: tuxpusher
+rename-desktop-file: tuxpusher.desktop
+rename-appdata-file: tuxpusher.appdata.xml
```

Comment:
You use the install command, to install this files. You can directly install them under the right name, so you don't need this.